Makefile

```
@@ -39,7 +39,7 @@ KONG_NETTLE_VERSION ?= `grep KONG_NETTLE_VERSION $(KONG_SOURCE_LOCATION)/.requir
 KONG_NGINX_MODULE ?= `grep KONG_NGINX_MODULE $(KONG_SOURCE_LOCATION)/.requirements | awk -F"=" '{print $$2}'`
 LIBYAML_VERSION ?= `grep LIBYAML_VERSION $(KONG_SOURCE_LOCATION)/.requirements | awk -F"=" '{print $$2}'`
 OPENRESTY_PATCHES ?= 1
-DOCKER_KONG_VERSION = 'master'
+DOCKER_KONG_VERSION = 'fix/rhel8'
```

putting a reminder here to swap this back to master after https://github.com/Kong/docker-kong/pull/438 merges

```suggestion
DOCKER_KONG_VERSION = 'master'
```